modules/install-secured-cluster-operator.adoc

```diff
@@ -14,29 +20,50 @@ To install Collector on systems that have Unified Extensible Firmware Interface
 ====
 
 .Prerequisites
-* You must be using {ocp} 4.6 or later.
-* You must have generated an init bundle and already created the required resources by using the init bundle.
+* If you are using {ocp}, you must install version 4.6 or later.
+* You have installed the {product-title-short} Operator.
+* You have generated an init bundle and already created the required resources by using the init bundle.
 
 .Procedure
 . On the {ocp} web console, navigate to the *Operators* -> *Installed Operators* page.
 . Select the {product-title} Operator from the list of installed operators.
-. By default, {ocp} lists the project as `rhacs-operator`. Select *Project: rhacs-operator* -> *Create project*.
+.. Select the *Installation mode* of *All namespaces on the cluster (default)*. 
+.. Under *Installed Namespace*, select the default project of `rhacs-operator`. If this namespace does not exist, the installation program creates it.
 +
 [WARNING]
 ====
 You must install {product-title} `SecuredCluster` resource in its own project and not the default `openshift-operators` project.
 ====
-. Enter the new project name as *stackrox* or some other name, and click *Create*.
-. Under the *Provided APIs* section, select *Secured Cluster*.
-. Choose *Create SecuredCluster*.
-. Enter a name for your `SecuredCluster` custom resource.
-. For *Central Endpoint*, enter the address and port number of your Central instance.
-For example, if Central is available at `\https://central.example.com`, then specify the central endpoint as `central.example.com:443`.
-The default value of `central.stackrox.svc:443` only works when you install secured cluster services and Central in the same cluster.
-. Accept the default values or configure custom values for the available options.
+.. Click *Install*. 
+. After the Operator is installed, click *View Operator*. 
+. Select *Secured Cluster* from the central navigation menu in the *Operator details* page.
+. Click *Create SecuredCluster*.
+. Select one of the following options in the *Configure via* field:
+* *Form view*: Use this option if you want to use the on-screen fields to configure the secured cluster and do not need to change any other fields. 
+* *YAML view*: Use this view to set up the secured cluster using the YAML file. The YAML file is displayed in the window and you can edit fields in it. If you select this option, when you are finished editing the file, click *Create*.
+. If you are using *Form view*, enter the new project name by accepting or editing the default value, *stackrox-secured-cluster-services*.
+. Optional: Add any labels for the cluster. ##Need more info about labels.##
```

Comment:
This note renders as do the rest. I'll leave this here since it's a few places

![image](https://user-images.githubusercontent.com/16167833/204658129-ddc6ad65-5f56-4b2c-9bf6-f171aa5eec1d.png)